README.md

```diff
@@ -0,0 +1,25 @@
+## PYCRAFTCO
+
+**pycraftco** is a Python library that serves as a convenient wrapper for [Craft API](https://api.craft.co/docs/v1)
+
+### Installation
+
+You can install package via `pip`:
+
+    $ pip install git+https://github.com/craft-machine/pycraftco.git
+
+
+### Usage
+
+```
+MINI TUTORIAL
+```
+
+
+### Contributing
+
+Bug reports and pull requests are welcome on GitHub at https://github.com/[USERNAME]/pycraftco.
```

Comment:
Please, replace the username with craft-machine.

Reply:
Replace it!